prow/resultstore/payload.go

```diff
@@ -136,9 +136,9 @@ func commandLines(pj *v1.ProwJob) []*resultstore.CommandLine {
 	if pj.Spec.PodSpec != nil {
 		for _, c := range pj.Spec.PodSpec.Containers {
 			cl = append(cl, &resultstore.CommandLine{
-				Label:   commandLineLabel,
-				Args:    c.Args,
-				Command: strings.Join(c.Command, " "),
+				Label: commandLineLabel,
+				Tool:  strings.Join(c.Command, " "),
```

Comment:
The commit message says `argv[0]`, but the code does `strings.Join(c.Command, " ")` which is a multi-word string. Should it just have the first word of `c.Command`?

Reply:
Making a note of this one as well. The (internal) prior art combined them, since command is a list for some reason. I don't know whether it's meant to contain sub-commands (e.g. ["git", "log"]. If that is the case, then we are meant to put the sub-command into Command, per my discussion with resultstore folks.

With this change, commands should at least render, and we can iterate.